templates/staff/mod/ban_confirm.php

```diff
@@ -13,13 +14,17 @@
         <td style="text-align: center;">
             <?php if($ban_length == StaffManager::PERM_BAN_VALUE): ?>
                 Issue permanent <?=ucwords($ban_type)?> Ban?
+            <?php elseif($ban_length >= StaffManager::MINUTES_PER_DAY * 30): ?>
+				<?=ucwords($ban_type)?> Ban <?=$user_data['user_name']?> for <?=($ban_length / (30 * StaffManager::MINUTES_PER_DAY))?> month(s)?<br />
```

Comment:
Consider adding another constant for this just to make life a little easier
```suggestion
				<?=ucwords($ban_type)?> Ban <?=$user_data['user_name']?> for <?=($ban_length / StaffManager::MINUTES_PER_MONTH)?> month(s)?<br />
```